src/main/java/in/org/projecteka/clientregistry/repository/OrganizationRepository.java

```diff
@@ -99,4 +102,39 @@ public Organization find(String id) {
         }
         return null;
     }
+
+    public int addOrUpdateOrganization(BridgeServiceRequest organization) {
+        Organization org = find(organization.getId());
+        if (org == null) {
+            return jdbcTemplate.update(
+                    "INSERT INTO organization(org_id, org_name, org_alias, org_type, active, phone, city, state," +
+                            " date_created, date_modified)" +
+                            " VALUES(?,?,?,?,?,?,?,?,?,?)"
+                    , organization.getId(), organization.getName(), String.join(",", organization.getOrgAlias()),
+                    organization.getType(), organization.getActive(), organization.getPhone(), organization.getCity(),
+                    organization.getState(), LocalDateTime.now(ZoneOffset.UTC), LocalDateTime.now(ZoneOffset.UTC));
+        } else {
+            String updateOrganizationQuery = "UPDATE organization SET ";
+            if (organization.getName() != null) {
+                updateOrganizationQuery += "org_name = '" + organization.getName() + "', ";
+            }
+            if (organization.getOrgAlias() != null) {
+                updateOrganizationQuery += "org_alias = '" + String.join(",", organization.getOrgAlias()) + "', ";
+            }
+            if (organization.getActive() != null) {
+                updateOrganizationQuery += "active = '" + organization.getActive() + "', ";
```

Comment:
Could you please confirm if giving boolean values in  single quotes(`''`) work. I suspect it would treat them as string.

Reply:
It's stored as a boolean value in DB. If we pass `true` or `false` in string format also it will do parsing and stores as boolean in the database. Otherwise throws parsing error while deserialization.

Reply:
Sure then! Thanks for the confirmation